pyproject.toml

```diff
@@ -48,6 +47,7 @@ isort = "^5.10.1"
 safety = "^1.10.3"
 mypy = "^0.942"
 coverage = "^6.3.2"
+volttron-testing = "^0.3.1a4"
```

Comment:
The version hasn't changed. Shouldn't it be [0.3.1a7](https://pypi.org/project/volttron-testing/)?